ui/styles/theme.go

```diff
@@ -18,23 +24,43 @@ type Theme struct {
 	SubleMainText      lipgloss.AdaptiveColor
 }
 
-var subtleIndigo = lipgloss.AdaptiveColor{Light: "#5A57B5", Dark: "#242347"}
+var DefaultTheme = func() Theme {
```

Comment:
The config file needs to be read to constuct the theme from it, and I couldn't come up with a sensible way to do that which also allows the `styles` package to keep working the way it does. I'm new to Go, so maybe I'm missing an obvious solution that'd keep the package working without rewriting everything in `ui/components`... how would you do it?

Reply:
What put me off that was that you'd then implicitly rely on something else to already have initialized it. I originally did it similar to that, and called the theme init function from the config parser, but then had to realize that due to the internal workings of lipgloss, a lot of colors weren't used and things defaulted to white instead, because parts of the lipgloss thing are used before the config is read.

We could add a non-exported variable to hold the theme, I think right now the config file is read every time `DefaultTheme` is referenced from other modules..? Not sure. As a cache of sorts, if that's the case.

Reply:
https://github.com/dlvhdr/gh-dash/pull/146/commits/25084f017a5c57acea16fa08fd054c8ae3488396

Reply:
Not sure if the function will be called multiple times or just evaluated once.
But cool that you added it.